pkg/rewards/operatorAvsSplitSnapshots.go

```diff
@@ -0,0 +1,92 @@
+package rewards
+
+const operatorAvsSplitSnapshotQuery = `
+WITH operator_avs_splits_with_block_info as (
+	select
+		oas.operator,
+		oas.avs,
+		oas.activated_at::timestamp(6) as activated_at,
+		oas.new_operator_avs_split_bips as split,
+		oas.block_number,
+		oas.log_index,
+		b.block_time::timestamp(6) as block_time
+	from operator_avs_splits as oas
+	left join blocks as b on (b.number = oas.block_number)
```

Comment:
I know this is my fault for doing it elsewhere, but this should probably be just a `join` so that we require the block record to exist. Functionally, it probably doesnt change much, but doing a left join allows the block info to potentially be null

Reply:
Done in [64bf833](https://github.com/Layr-Labs/sidecar/pull/117/commits/64bf8334c40fcf4505f74672a8341a96cd9858a2)